### ISSUE TITLE
Links open in _self rather than _blank when using Marked as markdown renderer

### ISSUE THREAD

**Issue by cjhille** [RocketChat/Rocket.Chat#20460]
### Description:

When using `marked` as markdown parsers, links in chat messages that contain a protocol (e.g. `http://` or `https://`) will open in the same window as rocket.chat, while links without a protocol will open in a new tab. 
This leads to issues when rocketchat is embedded in a frame etc. 
AFAIK this used to work in 3.9.1.

### Steps to reproduce:

1. set `marked` MD parser
2. post url in chat containing `http://`
3. click on url

### Expected behavior:

new tab should open

### Actual behavior:

link opens in current tab and user quits rocket.chat

### Server Setup Information:

- Version of Rocket.Chat Server: 3.10.5
- Deployment Method: docker

### Client Setup Information

- Desktop App or Browser Version: chrome

In DOM this will be the resulting link

```
<p>
    <a href="http://example.org" target="_blank" rel="noopener noreferrer">                example.org</a>
    <a href="http://example.org"                 rel="nofollow noopener noreferrer">http://example.org</a>
</p>
```

**Comment by sushant52** [comment]
Issue is not reproducible in 3.11.0-develop.

**Comment by cjhille** [comment]
Could it be that this PR (https://github.com/RocketChat/Rocket.Chat/pull/20435) from yesterday accidentally (or deliberately) fixed the broken message links as well?

**Comment by danel1** [comment]
Same problem here with 3.10.5 @ggazzo @sampaiodiego

**Comment by sushant52** [comment]
> Could it be that this PR (#20435) from yesterday accidentally (or deliberately) fixed the broken message links as well?

Sorry, the issue still persists.

**Comment by ggazzo** [comment]
I hope this issue no longer exist on develop branch 🙏

**Comment by danel1** [comment]
@ggazzo could you please backport it to 3.10.x? We do not want to upgrade to the first versions of 3.11.x for stability reasons. Thanks!

**Comment by nickbroon** [comment]
This issues is also seen with the 3.9.7 release.

**Comment by cjhille** [comment]
> I hope this issue no longer exist on develop branch 🙏
@ggazzo 
Unfortunately the issue still persists in 3.11.0, which contained the changes of the develop branch at the time of your comment.

**Comment by sdryga** [comment]
Hi!
I faced with this issue after upgrade to 3.10.5. Now I'm on 3.11.0 and problem still exists.
Please fix it, it's very annoying when http(s):// links open in the same window as rocket.chat
Thanks!

**Comment by andreas-monday** [comment]
the problem remains in the version 3.11.0

**Comment by bb2k** [comment]
I confirm this issue in 3.11 and it really annoying

**Comment by andreas-monday** [comment]
the problem continues in the version 3.11.1

**Comment by pierreozoux** [comment]
@ggazzo is it fixed? it is not clear :)

**Comment by ggazzo** [comment]
@pierreozoux it should be fixed long time ago, version 3.12.0
https://github.com/RocketChat/Rocket.Chat/pull/20651

**Comment by cjhille** [comment]
Can confirm. The described issue is fixed.

**Comment by b90g** [comment]
Seems to be fixed in RC 6.0.0
